src/authenticationinapp/authenticationinapp.cpp

```diff
@@ -90,6 +90,13 @@ void AuthenticationInApp::signIn() {
   m_listener->signIn();
 }
 
+QString AuthenticationInApp::emailAddress() {
+  Q_ASSERT(m_state == StateSignIn);
```

Comment:
I would remove this assert. In theory, you can ask the emailAddress at any time, right?

Reply:
done